merged_schema/HPXMLMerged.xsd

```diff
@@ -4894,6 +4894,69 @@
 					<xs:attribute name="dataSource" type="DataSource"/>
 				</xs:complexType>
 			</xs:element>
+			<xs:element minOccurs="0" name="MeasureType">
+				<xs:complexType>
+					<xs:choice>
+						<xs:element name="InstallReplace">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element ref="extension" minOccurs="0"/>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element name="AtticEncapsulation">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="DuctsMovedToConditionedSpace" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" ref="extension"/>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element name="FoundationEncapsulation">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="DuctsMovedToConditionedSpace" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" ref="extension"/>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element name="HVACRedesign">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="DistributionTypeChange" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" name="PartialDisplacement" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" name="HotWaterIntegrated">
+										<xs:simpleType>
+											<xs:restriction base="xs:string">
+												<xs:enumeration value="separated"/>
+												<xs:enumeration value="integrated"/>
+											</xs:restriction>
+										</xs:simpleType>
+									</xs:element>
+									<xs:element minOccurs="0" ref="extension"/>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element name="DuctImprovement">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="DuctInsulation" type="HPXMLBoolean">
+										<xs:annotation>
+											<xs:documentation>Add ids to the duct insulation (breaking change)?</xs:documentation>
+										</xs:annotation>
+									</xs:element>
+									<xs:element minOccurs="0" name="DuctSealing" type="HPXMLBoolean">
+										<xs:annotation>
+											<xs:documentation>Add ids to the duct sealing or leakage (breaking change)?</xs:documentation>
```

Comment:
@nmerket It looks like this is a reminder to consider adding IDs (a breaking change). Should we just delete these comments for now and consider this change for a future HPXML version?

Reply:
Do you remember the context for this @GamalielL?

Reply:
This was about adding IDs to the DuctInsulation and DuctLeakage sections so that they could be referenced as installed/replaced components.  This would make the DuctImprovement choice unnecessary, because duct insulation and sealing measures could be distinguished by the referenced components.

Reply:
Should we make that change or leave it as is with the DuctImprovement choice?

Reply:
Can we make a decision here so we can move forward? I vote for leave as is (by which I mean removing the `xs:documentation` comments), mostly because I think we want to avoid a breaking change. But I don't have a strong opinion.